multinode-demo/start_nodes.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+nodes_file=$1
+remote_user=$2
+ssh_keys=$3
+
+oldIFS="$IFS"
+IFS=$'\n' ip_addr_array=($(<$nodes_file))
+IFS="$oldIFS"
```

Comment:
Save/restore IFS shouldn't be needed.  Line 8 only changes IFS for that one assignment...i think